calgroup_builder/__init__.py

```diff
@@ -0,0 +1,399 @@
+import asyncio
+import json
+import logging
+import os
+from functools import partial
+from textwrap import dedent
+from urllib.parse import quote
+
+from packaging.version import Version as V
+from tornado.httpclient import AsyncHTTPClient, HTTPRequest
+from tornado.httputil import url_concat
+from tornado.ioloop import IOLoop, PeriodicCallback
+from tornado.log import LogFormatter
+from traitlets import Bool, Int, Unicode, default
+from traitlets.config import Application
+
+import subprocess
```

Comment:
subprocess is a native python library so I'd move the import to the top section after `import os`.